packages/client/wallets/aa/package.json

```diff
@@ -27,6 +27,7 @@
     },
     "dependencies": {
         "@ambire/signature-validator": "1.3.1",
+        "@crossmint/client-sdk-base": "file:../../crossmint-sdk/packages/client/base",
```

Comment:
this should be the release from https://github.com/Crossmint/crossmint-sdk/pull/334